BOOTSEQ.md

```diff
@@ -4,77 +4,160 @@
 
 Entities optionally participating in the boot/reboot/shutdown processes:
 
-- Platform UEFI/BIOS of the Server
-- Platform BMC of the Server
+- Platform UEFI/BIOS of the host
+- Platform BMC of the host
 - xPU BMC
 - xPU ATF/UEFI/BIOS
-- additional xPUs that reside within the Server
+- additional xPUs that reside within the host
 
 ## Assumptions
 
-- Resetting the xPU is assumed to cause PCIe errors that will crash the server (at least for some scenarios).
-  - This will cause challenges for deploying an OS on the xPU and when running multi-xPU deployments. Perhaps OPI should work with Linux Foundation to encourage host and OS developers to properly manage the response by the host and the xPU(s) from a Reset
-  - There are different kinds of resets. Genrally XPU core complex reset and XPU PCIe MAC reset should be separated for minimal disruptive ISSU etc. but a complete SoC reset of the xPU will cause PCIe errors and needs reboot the server.
-- There are use cases for network boot only.
+- Resetting the xPU is assumed to cause PCIe errors that may impact the operation of the host OS.
+  - There are different kinds of resets. Generally XPU core complex reset and XPU PCIe MAC reset should be separated for minimal disruptive ISSU etc. but a complete SoC reset of the xPU will cause PCIe errors and may result in a controlled or uncontrolled host reset.
```

Comment:
Actually the challenge occurs when the ARM cores controll the PCIe layer.  During the time it takes for the PCIe layer to initialize, the xPU is likely to cause PCIe errors for PCIe Config Cycles.

Reply:
The "complete SoC reset" statement is the main line here, IMHO. I would be inclined to assume that no matter what, cold/warm full xPU reset domain incorporates both PCIe subsystem and Arm complex, no matter if PCIe subsystem is completely independent or tied to Arm complex, as @jainvipin distinguishes. As a result, PCIe errors caused by that reset may result in host reset, controllable or not.

Downstream from there, we could either state an OPI preference or advisory that xPU design might/should draw separate reset domains for PCIe subsystem and Arm complex, because of <insert appropriate> reasons. However, it might be risky, considering the possible relationship between Arm complex and PCIe subsystem. To add more flavor, in Intel IPUs, IMC complex programs PCIe subsystem and handles resets coming from the host (e.g. PERST#); ACC complex is not involved.

Reply:
> Actually the challenge occurs when the ARM cores controll the PCIe layer. During the time it takes for the PCIe layer to initialize, the xPU is likely to cause PCIe errors for PCIe Config Cycles.

agree, if ARM cores are involved in controlling PCIe layer, then this would have an impact. If the programmability of the PCIe layer is controlled by the ARM complex, then too this would be indirectly involving ARM complex.